onnxruntime/test/providers/cpu/reduction/reduction_ops_test.cc

```diff
@@ -57,11 +57,15 @@ TEST(ReductionOpTest, ReductionVariationTest) {
       std::vector<int64_t> expected_values;
       for (auto v : std::get<2>(a.second))
         expected_values.push_back(static_cast<int64_t>(v));
-      TestReduceOp<int64_t>(a.first, input_dims, input_data, attributes.axes_, attributes.keep_dims_,
+      TestReduceOp<int64_t>(a.first, 7, input_dims, input_data, attributes.axes_, attributes.keep_dims_,
```

Comment:
It looks like NGraph pieplines are breaking on account of this. Maybe they have not been excluded for the negative axes case ? Line 28 looks like a suspect - 

"if (op.compare("ArgMax") == 0 || op.compare("ArgMin") == 0) {"

Should it have something like Reduce* ?